RFS/src/test/java/com/rfs/framework/SimpleRestoreFromSnapshot_ES_7_10.java

```diff
@@ -80,7 +80,8 @@ public void updateTargetCluster(
                 ).readDocuments();
 
                 final var finalShardId = shardId;
-                new DocumentReindexer(client, 1000, 1).reindex(index.getName(), documents, context)
+                new DocumentReindexer(client, 1000,
+                    Long.MAX_VALUE, 1, 3).reindex(index.getName(), documents, context)
```

Comment:
weird spacing?  Please don't wrap one function call and then put the next on the continued line unless it's a fluent .log() call.